diktat-test-framework/pom.xml

```diff
@@ -69,6 +69,9 @@
                             <goal>compile</goal>
                         </goals>
                         <configuration>
+                            <args>
+                                <arg>-Werror</arg>
```

Comment:
Why are these flags duplicated among modules? Other from that, I think everything is ready

Reply:
Fixed